app/lib/features/main/presentation/pages/main_page.dart

```diff
@@ -7,110 +7,242 @@ class MainPage extends StatefulWidget {
   State<MainPage> createState() => _MainPageState();
```

Comment:
please revert this entire page; this one is still broken 😅😅😅
<img width="373" alt="Screenshot 2023-10-17 at 20 48 49" src="https://github.com/ia-toki/bebras-pandai/assets/1970083/47a27887-ef08-419c-a9de-2d657425e6ac">